index.bs

```diff
@@ -70,11 +105,23 @@ The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientat
 Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
 of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
 
-A [=latest reading=] per [=sensor=] of orientation type includes one [=map/entries=]
+A [=latest reading=] per [=sensor=] of orientation type includes an [=map/entry=]
 whose [=map/key=] is "quaternion" and whose [=map/value=] contains a four element [=list=].
-The elements of the [=list=] are equal to components of a unit quaternion representing orientation
-of the device in a three dimensional Cartesian coordinate system (x, y, z), where x-axis points east,
-y-axis points north and z-axis points up and is perpendicular to the plane made up of x and y axis.
+The elements of the [=list=] are equal to components of a unit quaternion [[QUATERNIONS]]
+[cos(θ/2), V<sub>x</sub> * sin(θ/2), V<sub>y</sub> * sin(θ/2), V<sub>z</sub> * sin(θ/2)] where V is the unit vector representing
+the axis of rotation. The quaternion represents rotation of a device hosting motion sensors in relation to a <dfn>Earth's reference coordinate system</dfn> defined as a three
```

Comment:
Maybe say that the first component is the scalar representing the actual rotation angle